src/brand.md

```diff
@@ -67,7 +67,11 @@ rules governing the proper use of the Flutter trademarks.
   trademarks of Google LLC. We are not endorsed by or affiliated with
   Google LLC."
 * Keep some distance between the Flutter trademarks and any other trademarks,
-  logos, or icons that are displayed on the webpage.
+  logos, or icons that are displayed on the webpage.  
+* <b>[Unofficial Flutter Events]</b> You may use the Flutter trademarks as part of the name 
```

Comment:
This feels like it might be better under "Community Use Exceptions", since it's a similar rule to the one about local user groups. WDYT?

Reply:
I’m waiting for you and @timsneath to resolve.

Reply:
Ok, initially thought to include in DO as it's more visible, but if we bold the [Unofficial Flutter Events] then the community section works. I'll adjust